modules/cnf-performing-platform-verification-latency-tests.adoc

```diff
@@ -27,18 +24,37 @@ Validations include:
 * Enabling the `contain-mount-namespace` mode via machine config
 * Using OVN-kubernetes as the cluster network provider
 
-Latency tests, a part of the CNF-test container, also require the same validations. For more information about running a latency test, see the Running the latency tests section.
+[NOTE]
+====
+The CNF tests image includes several tests that are not supported by Red Hat at this time. Only the latency tests are supported.
+====
 
-The tests need to perform an environment configuration every time they are executed. This involves items such as creating SR-IOV node policies, performance profiles, or PTP profiles. Allowing the tests to configure an already configured cluster might affect the functionality of the cluster. Also, changes to configuration items such as SR-IOV node policy might result in the environment being temporarily unavailable until the configuration change is processed.
+[IMPORTANT]
+====
+When running the latency tests, **always** run the tests with `DISCOVERY_MODE=true`. For example:
+
+[source,terminal]
+----
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
+DISCOVERY_MODE=true registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
```

Comment:
This will run the full set of tests. We should be specific about latency tests

Reply:
I've updated this and the rest. We'll need to walk through this on a call I think.